### DISCUSSION TITLE
How to Choose Activation Functions

### DISCUSSION THREAD

**Discussion by kkmcgg** [NSCC-COGS/Aestheta#65] [Guides]
https://keras.io/api/layers/activations/

Activation functions are magical little equations we apply just before the output of a given neuron to restrict and control the possible range of values after all the neurons inputs have been individually multiplied by some weight and all summed together. If neurons were like lightbulbs with many many little wires with various amounts of electricity plugged into them, the activation function would control the 'curve' by which the lightbulb gets brighter as the incoming electricity changes. 

The basic TLDR or this guide, and general advice you'll see again and again is '**_use relu_**' though in realty there are many instances where it maybe worthwhile to try a _**sigmoid**_ activation function or to experiment with others.

**Comment by kkmcgg** [comment]
This write up may also be useful to some 
https://medium.com/the-theory-of-everything/understanding-activation-functions-in-neural-networks-9491262884e0
